db_schema_diagram.sql

```diff
@@ -0,0 +1,51 @@
+CREATE DATABASE clinic;
+CREATE TABLE patients (
+  id SERIAL PRIMARY KEY,
+  name VARCHAR(100) NOT NULL,
+  date_of_birth DATE NOT NULL
+);
+
+CREATE TABLE medical_histories (
+  id SERIAL PRIMARY KEY,
+  admitted_at TIMESTAMP,
+  patient_id INT NOT NULL,
+  status VARCHAR(150)
+);
+CREATE TABLE treatments (
+    id SERIAL PRIMARY KEY,
+    type VARCHAR(255),
+    NAME VARCHAR(255)
+);
```

Comment:
- [ ] Kindly note that there is a many-to-many relationship between the `medical_histories` and `treatments` tables. Therefore it would be nice to have a **JOIN TABLE** `medical_histories_treatments` reflecting the **many-to-many** relationship with `medical_history_id` and `treatment_id` as foreign keys referencing the `medical_histories` and `treatments` tables respectively